components/shared/searchInput/SearchInput.tsx

```diff
@@ -16,36 +18,34 @@ type SearchInputProps = {
 const SearchInput = ({ className, placeholder }: SearchInputProps) => {
   const inputRef = useRef<HTMLInputElement>(null)
   const router = useRouter()
-  const searchParams = useSearchParams()
-  const [queryString, setQueryString] = React.useState("")
+  const actor = useSearchMachineActor()
+  const currentQ = useSelector(actor, snapshot => {
```

Comment:
I dont like abbreviation / shortforms. Q could be hard to figure out what it is exactly. currentQuery is a much more descriptive name IMO.

In general i like descriptive code and variable names.

```suggestion
  const currentQuery = useSelector(actor, snapshot => {
```

Reply:
👍